src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -477,13 +499,15 @@ void DisposeRelatedPullReplication(string hub, string certThumbprint)
         public void RunPullReplicationAsSink(TcpConnectionOptions tcpConnectionOptions, JsonOperationContext.MemoryBuffer buffer, PullReplicationAsSink destination)
         {
             string[] allowedPaths = DetailedReplicationHubAccess.Preferred(destination.AllowedHubToSinkPaths, destination.AllowedSinkToHubPaths);
-            var pullParams = new IncomingPullReplicationParams
+            var incomingPullParams = new PullReplicationParams
             {
                 Name = destination.HubName,
                 AllowedPaths = allowedPaths,
-                Mode = PullReplicationMode.HubToSink
+                Mode = PullReplicationMode.HubToSink,
+                PreventDeletionsMode = null, // Not yet configured because this is before getting a reply from hub
```

Comment:
Where is this being set?

Reply:
I don't actually bother to set this because the flag is irrelevant in HubToSink mode. The comment is kinda misleading so I took it off